packages/Select/index.tsx

```diff
@@ -0,0 +1,394 @@
+import React, { forwardRef, Fragment, useEffect, useMemo, useState } from 'react'
+import Downshift, { ControllerStateAndHelpers, DownshiftProps } from 'downshift'
+import { matchSorter } from 'match-sorter'
+import { DownIcon } from '@welcome-ui/icons.down'
+import { ClearButton } from '@welcome-ui/clear-button'
+import { createEvent, Variant } from '@welcome-ui/utils'
```

Comment:
I add on FileDrop PR CreateEvent type, maybe you can reuse it on this PR

Reply:
https://github.com/WTTJ/welcome-ui/pull/956/files#diff-625a73dee767d8acfd76b9ac6962f40ece0372c19150f412dbce4a7665c0ef40

Reply:
✅